views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -57,11 +61,15 @@ define([
             $counterMaxLength = $('.text-counter-chars > .count-max-length', $original);
 
         var formats = {
-            plain : {label : __("Plain text"), selected : false},
-            preformatted : {label : __("Pre-formatted text"), selected : false},
-            xhtml : {label : __("XHTML"), selected : false}
+            plain : {label : __('Plain text'), selected : false},
+            preformatted : {label : __('Pre-formatted text'), selected : false},
+            xhtml : {label : __('Rich text'), selected : false}
         };
 
+        if (config.hasMath) {
+            formats.math = {label : __('Rich text') + ' + ' + __('math'), selected : false};
```

Comment:
**nitpick:** Why not having a complete sentence instead? This may introduce translation issues, not all languages are following the same word order. We must avoid using composed translation sentences unless they serve different and separate meanings.

**suggestion:** Wouldn't this be better?
```suggestion
            formats.math = {label : __('Rich text + math'), selected : false};
```